ceph_installer/tests/controllers/test_agent.py

```diff
@@ -29,8 +29,9 @@ def test_index_post_is_denied(self, session, monkeypatch):
             expect_errors=True,
             extra_environ=dict(REMOTE_ADDR='192.168.1.1')
         )
-        assert result.status_int == 403
-        assert result.json['message'] == 'this resource does not allow non-local requests'
+        assert result.status_int == 200
```

Comment:
Looks like we're removing the `403` test entirely here? Can you create a separate test, to ensure only `/api/agent/` is allowed?